network-api/networkapi/wagtailpages/customblocks.py

```diff
@@ -2,6 +2,16 @@
 from wagtail.images.blocks import ImageChooserBlock
 
 
+class AnchorBlock(blocks.StructBlock):
+    fragmentid = blocks.CharBlock(
+        help_text='Use this to place an explicit fragmentid on the page for linking purposes.'
```

Comment:
This text is too technical. Use something like `The text entered here will enable direct linking to this part of the page, using a # before it. For example, "foo" allows you to use <a href="#foo">`

Reply:
that's a very long text for a tiny, tiny block. Can we condense that? (maybe refer to somewhere in the "how do I wagtail"?)

Reply:
Sure. Not a "use this" just a "like this" suggestion